src/control/Controlador.py

```diff
@@ -48,6 +48,11 @@ def __init__(self, v_saida=100, v_retorno=100):
         print('volume_saida = ', self.volume_saida)
         print('volume_retorno = ', self.volume_retorno)
 
+    def alterar_som(self, alt_volume_saida, alt_volume_retorno):
```

Comment:
```suggestion
    def alterar_volume(self, alt_volume_saida, alt_volume_retorno):
```
A finalidade do método fica mais explícita assim.